apps/heft/src/configuration/HeftConfiguration.ts

```diff
@@ -161,11 +160,12 @@ export class HeftConfiguration {
     );
     if (packageJsonPath) {
       let buildFolderPath: string = path.dirname(packageJsonPath);
-      // The CWD path's casing may be incorrect on a case-insensitive filesystem. Some tools, like Jest
-      // expect the casing of the project path to be correct and produce unexpected behavior when the casing
-      // isn't correct.
-      // This ensures the casing of the project folder is correct.
-      buildFolderPath = trueCasePathSync(buildFolderPath);
+      // On Windows it is possible for the drive letter in the CWD to be lowercase, but the normalized naming is uppercase
+      // Force it to always be uppercase for consistency.
+      buildFolderPath =
+        process.platform === 'win32'
+          ? buildFolderPath.charAt(0).toUpperCase() + buildFolderPath.slice(1)
```

Comment:
Windows also supports paths in the form of `\\?\C:\folder\file.wsdl`. See https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats

Reply:
While that's true, in practice we don't expect to receive paths in that format. They get converted to that inside of the file system calls, though.